src/app/modules/color-scheme/color-scheme-contents/color-scheme-contents.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="color-scheme-contents-container">
+  <div id="scheme-contents" class="scheme-contents">
+    <div class="swatch one" (click)="schemeChanged(0)">
```

Comment:
Create a ColorBar component capable of displaying a color scheme (discrete or gradient) then replace all swatches with a ngFor loop.
I.e.
`<ccf-color-bar *ngFor="scheme of colorSchemes" ...></...>`